x/lscosmos/keeper/reset_data_test.go

```diff
@@ -125,13 +126,6 @@ func (suite *IntegrationTestSuite) TestFork() {
 
 	suite.Require().NoError(simapp.FundAccount(pstakeApp.BankKeeper, ctx, authtypes.NewModuleAddress(types.UndelegationModuleAccount), sdk.NewCoins(stkatomswithModule)))
 
-	err := k.ClaimFailed(ctx, types.DelegatorUnbondingEpochEntry{
```

Comment:
this is required... till this line it is an effort to get  to mainnet state.. since 232 epoch user claimed failed unbondings.